### PR TITLE
feat(project-page): init template - INNO-412

### PR DESCRIPTION
**Pull request by emeryro**
Template of project page.
Also includes some utilities.

### CODE REVIEW

**Comment by emeryro** [comment]
Modifications done

File: framework/components/contracts/_contracts.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
   max-width: 20em;
   text-align: center;
 
-  &__image {
+  &__image img {
```
Comment (kalinchernev) [inline framework/components/contracts/_contracts.scss]: I think you can simplify this:
- apply the __image class directly on the image
- provide only the src attribute in the config.js

Or another suggestion would be to make a more generic class which can be applied to any image, not only the image within the contract. The current implementation is not wrong, but it requires more thinking and gymnastics in implementation.

File: framework/templates/project-page/project-page.twig [hunk 2]
```diff
@@ -1,4 +1,234 @@
...
+
+          <p><strong>Main objective:</strong><br />
+            Contribute to the improvement of food and nutrition security of very poor houseohlds in the Matam region. Improve access, availability and diversity of food products in rural areas.</p>
+        </div>
+      </div>
+    </div>
+  {% endblock %}
+{% endembed %}
+
+{% embed "@sections--map" %}
```
Comment (kalinchernev) [inline framework/templates/project-page/project-page.twig]: maps_description

No idea if we have control over this, but it's really taking the nice design down ....

File: framework/utilities/colors/colors.config.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 module.exports = {
   title: 'Colors',
   label: 'Colors',
+  status: 'wip',
```
Comment (kalinchernev) [inline framework/utilities/colors/colors.config.js]: Why in progress?

File: framework/components/contracts/_contracts.scss (hunk 1 above)
Comment (emeryro) [inline framework/components/contracts/_contracts.scss]: Well, in that case we should also alt text and maybe width and height in config.js
But I think your suggestion to have utilities for this is the best: one class to make image round, and spacing class already available

File: framework/templates/project-page/project-page.twig (hunk 2 above)
Comment (emeryro) [inline framework/templates/project-page/project-page.twig]: Yes, I noticed it too. I'm not sure we have control over this, but I created another task to handle this

File: framework/utilities/colors/colors.config.js (hunk 3 above)
Comment (emeryro) [inline framework/utilities/colors/colors.config.js]: Indeed, we can consider it done on our side.
We didn't get formal approuval of colors yet, but we would open a new task if needed.

**Comment by emeryro** [comment]
Modifications applied

**Pull request merged**